### ISSUE TITLE
Copying -> Pasting text from external sources sometimes does not work in editor

### ISSUE THREAD

**Issue by istinnstudio** [apache/netbeans#5923]
### Apache NetBeans version

Apache NetBeans 17

### What happened

Copying and pasting text from other apps like a web browser sometimes does not work, it works only after an other different kind of copy will be triggered and will take place like a copy of a class in projects tab, after this I can copy-paste again for external sources until the next "internal" editor copy/paste that will re introduce the issue,

The issue is always present, I cannot think of any external interventions in between.

### How to reproduce

I am not sure if this is reproducible in other setups, I think it happens if a text copy from the internal editor will take place after or before a copy from an external source.

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

windows 10

### JDK

17

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

**Comment by neilcsmith-net** [comment]
Duplicate of #3962 

Please try some of the suggestions there, and help us figure out what the cause and best solutions are. Thanks!
